app/frontend/packs/providers-autocomplete.js

```diff
@@ -0,0 +1,29 @@
+import accessibleAutocomplete from "accessible-autocomplete";
+
+const initProvidersAutocomplete = () => {
+  try {
+    [
+      "#candidate-interface-pick-provider-form-code-field",
+      "#candidate-interface-pick-provider-form-code-field-error"
+    ].forEach(id => {
+      const providersSelect = document.querySelector(id);
+
+      if (!providersSelect) return;
+
+      // Replace "Select a provider" with empty string
+      providersSelect.querySelector("[value='']").innerHTML = "";
```

Comment:
I still see this (as part of the un-enhanced form control) for an instant while the JS loads. Is there a better way? (Maybe `visiblity: hidden` on the autocomplete until it's loaded?)

Reply:
This is part of the courses autocomplete too. Added tech debt card here: https://trello.com/c/hJwqQPsX/398-boilerplate-to-set-up-valid-autocompletes